src/Psalm/Internal/Codebase/ClassLikes.php

```diff
@@ -1811,14 +1811,14 @@ private function checkMethodReferences(ClassLikeStorage $classlike_storage, Meth
                                         true
                                     );
                                 }
-                            } elseif (IssueBuffer::accepts(
-                                $issue,
-                                $method_storage->suppressed_issues,
-                                $method_storage->stmt_location
+                            } else {
+                                IssueBuffer::maybeAdd(
+                                    $issue,
+                                    $method_storage->suppressed_issues,
+                                    $method_storage->stmt_location
                                     && !$declaring_classlike_storage->is_trait
                                     && !$has_variable_calls
```

Comment:
Indenting the subsequent lines would make it easier to see it's a part of the single expression